tests/unit/utils/test_standarddir.py

```diff
@@ -326,12 +332,16 @@ def test_system_datadir_exist_linux(self, monkeypatch):
         assert standarddir.system_data() == "/usr/share/qutebrowser"
 
     @pytest.mark.linux
-    def test_system_datadir_not_exist_linux(self, monkeypatch):
+    def test_system_datadir_not_exist_linux(self, monkeypatch, tmpdir):
         """Test that system-wide path isn't used on linux if path not exist."""
+        args = types.SimpleNamespace(basedir=str(tmpdir))
+        standarddir.init(args)
```

Comment:
With the `qtwebengine` branch I also added a `fake_args` fixture so you can do `fake_args.basedir = str(tmpdir)` (similar to `config_stub`) and it also gets registered in `objreg` accordingly - would you mind rebasing and switching to that?